addons/sale/static/src/js/sale_product_field.js

```diff
@@ -123,6 +123,9 @@ export class SaleOrderLineProductField extends ProductLabelSectionAndNoteField {
         return false;
     }
     get isConfigurableTemplate() {
+        if (!this.props.record.data.product_template_id) {
+            return;
+        }
         return this.props.record.data.is_configurable_product;
```

Comment:
```suggestion
        return this.props.record.data.product_template_id && this.props.record.data.is_configurable_product;

```
Do this in single line